.github/ci_configs/vllm.yaml

```diff
@@ -0,0 +1,24 @@
+external_build:
+  name: vllm-sm80-external-build
+  build_target: vllm
+  torch_whl_dir: dist
+  work_directory: vllm
+  # if set, overrides the base image used in vllm build
+  base_image: ${DOCKER_IMAGE} # replace with the env_var DOCKER_IMAGE
+  # if set, replaces the vllm dockerfile.torch_nightly with the local dockerfile.
+  dockerfile_path: "./.github/docker/Dockerfile.tmp_vllm"
```

Comment:
Nit:  Let's maintain some structure to the `ci_configs` directory.  For example,

```
.
└── ci_configs/
    └── vllm (the build target)/
        ├── config.yml (the config of the build target)
        └── docker (the associated Dockerfile)/
            └── Dockerfile
````